scripts/o3de/ExportScripts/export_source_built_project.py

```diff
@@ -229,29 +226,13 @@ def export_standalone_project(ctx: exp.O3DEScriptExportContext,
                                             archive_output_format=archive_output_format,
                                             logger=logger)
         if build_config == 'profile':
-            generate_launcher_startup_script(selected_platform, ctx.project_name, export_layout.launcher_type, export_layout.launch_flags, export_layout.output_path)
-
-def generate_launcher_startup_script(platform, project_name, launcher_type, launch_flags, output_path):
-    launcher_name = 'GameLauncher' if launcher_type == exp.LauncherType.GAME else \
-                    ('ServerLauncher' if launcher_type == exp.LauncherType.SERVER else 'UnifiedLauncher')
-    slash = '.\\' if platform == 'pc' else './'
-
-    # We do not use pathlib in this specific instance b/c its string conversion removes the initial './', which can trip up the shell script
-    exec_path = f"{slash}{project_name}.{launcher_name}{'.exe' if platform == 'pc' else ''}"
-    script_extension = 'cmd' if platform == 'pc' else 'sh'
-    comment_type = 'REM' if platform == 'pc' else '#'
-    cmd_text = f"""
-{comment_type} This game was exported using profile mode.
-{comment_type} Profile games try to open the AssetProcessor by default, but PAK files must be used for portability, so disable connecting to asset processor
-{comment_type} Use a release mode configuration to avoid having to use this script
-
-{str(exec_path)}  {' '.join(launch_flags)}
-"""
-    output_path.mkdir(parents = True, exist_ok=True)
-    script_path = output_path / f'{project_name}.{launcher_name}.START_GAME.{script_extension}'
-    
-    with open(script_path, 'w+') as cmd_file:
-        cmd_file.write(cmd_text)
+            generate_launcher_startup_setreg_patch(ctx.project_name, export_layout.output_path / 'Registry')
+
+def generate_launcher_startup_setreg_patch(project_name, output_path):
+    setregpatch = [{ "op": "add", "path": "/O3DE/Autoexec/ConsoleCommands/bg_ConnectToAssetProcessor", "value": False }]
+    output_path.mkdir(parents=True, exist_ok=True)
+    with open(output_path / f'{project_name.lower()}_export.setregpatch', 'w+') as patch_file:
```

Comment:
This works? I thought you needed to have the setreg name to include `*.client*.` in order for it to work with just the client/game launcher?

Reply:
_i_like_pie.setregpatch_ also works. If you include .game or .profile, then those keywords are respected so....
_i_like_pie.setregpath.release_ won't work when opening a profile launcher. 
Not sure it's worth adding .profile. in the name since we're only generating it for profile exports... but couldn't hurt actually. also makes it very explicit.

Note: I'm not sure setregpatch's respect .profile .release. I know setreg files do, but not sure about setregpatch. Would need to test that too.

Reply:
Also if this is meant just for profile it should include the `profile` tag as well.
i.e "launcher_export.client.profile.setreg" would only load when running the GameLauncher that was built in profile configuration.

Also this file doesn't need to be opened with 'w+' as it is not being read.

Reply:
@AMZN-Gene .setregpatch files are treated the same as .setreg files with the exception that the contents are parsed patched on top of the in-memory Settings Registry using JSON Patch instead of JSON Merge Patch.

All of the features between both are the same(i.e support for specialization tags, `$import` tag, etc...)

Reply:
like gene mentioned, the name is very flexible, we found out. If you have a `filename.setreg`, it'll pick up:
`filename.game.setreg` and `filename.game.profile.setreg`.